src/repos.json

```diff
@@ -467,6 +467,15 @@
       "forks": 0,
       "description": "Sync secrets from Credstash table to Kubernetes secret",
       "language": "Python"
+    },
+    {
+      "html_url": "https://github.com/razorpay/i18nify",
```

Comment:
do we want to add i18nify-react as well? https://github.com/razorpay/i18nify-react